src/utils/authFetch.js

```diff
@@ -10,15 +11,22 @@ export const authFetch = (url, data, errorMessage) => {
     const authData = Object.assign(data || {}, {headers: newHeaders})
     return checkedFetch(url, authData, errorMessage)
         .catch(httpError => {
-            if(httpError.code && httpError.code === 401)
+            if(httpError.code === 401)
```

Comment:
To się nie wywali na jakimś undefinied? Bo poprzedni if to sugerował XD

Reply:
Ten pierwszy warunek był bez sensu, bo jeśli `httpError.code` ma być 401, to `!!httpError.code` będzie zawsze true